### PR TITLE
Localizing For Nepal

### PR DESCRIPTION
**Pull request by swornim00**
Adding Nepali Language

### CODE REVIEW

File: locales/ne-NP.ftl [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+own-tweet-cta = आफ्नै शब्दमा ट्वीट गर्नुहोस 
+hashtag = #newfirefox
+github = GitHub
+legal = कानून
+license = अनुज्ञप्ति
+guidelines = दिशा निर्देश
+mozilla-description =
+  Mozilla एक नॉन-प्रॉफिट कंपनी हो जो जुटिरहेको छ तपाईंको online experience को रिमोट-कंट्रोल तपाईंलाई दिनको लागि र वेबको नक्सा मन्छेको हितको लागि परिवर्तन गर्न। बिस्तृत रुपमा जन्नकोलागि:
+  <a href="https://mozilla.org" target="_blank">mozilla.org</a>
+official-tweet-id = 930435170288656384
```
Comment (MichaelKohler) [inline locales/ne-NP.ftl]: Could you please update the ID to include the latest from the en-US file?

**Review by MichaelKohler — COMMENTED** [review]
Please see my comment. Rest looks good. Thanks a lot!

File: locales/ne-NP.ftl (hunk 1 above)
Comment (swornim00) [reply locales/ne-NP.ftl]: Done 😄

**Comment by swornim00** [comment]
Updated Official Tweet ID

**Review by MichaelKohler — APPROVED** [review]
Thanks!

**Pull request merged**